Marlin/Marlin.h

```diff
@@ -110,6 +110,8 @@ FORCE_INLINE void serialprintPGM(const char *str) {
 
 void get_command();
 void process_commands();
+uint16_t gcode_M238(uint8_t power = 0);
+void gcode_M241(long num_milliseconds = NULL);
```

Comment:
This is so M238 and M241 can be called from any other function/in a different scope. For example, when probing the bed level, M241 will be called to pause for a certain amount of time.\

But yeah, definitely a good idea about the `#ifdef` stuff.